exporter/awsemfexporter/grouped_metric_test.go

```diff
@@ -509,7 +550,11 @@ func TestTranslateUnit(t *testing.T) {
 	assert.Equal(t, "Count", v)
 }
 
-func generateTestMetricMetadata(namespace string, timestamp int64, logGroup, logStreamName, instrumentationScopeName string, metricType pmetric.MetricType) cWMetricMetadata {
+func generateTestMetricMetadata(namespace string, timestamp int64, logGroup, logStreamName, instrumentationScopeName string, metricType pmetric.MetricType, batchIndex ...int) cWMetricMetadata {
```

Comment:
I'm not a big fan of simulating optional parameters with variadic functions. I think it makes it harder to understand the meaning of the code. It encourages questions like "can I pass more than one parameter there? what happens if I do?". It makes it impossible for the compiler to report an issue when the function is mistakenly called with more than one "optional" parameter. Why not refactor the function to accept a regular `batchIndex` parameter? 

This is an internal function, so it's not a big problem.

Reply:
Thank you for the feedback! I’ve updated the function to use a regular batchIndex int parameter.